### ISSUE TITLE
BUG: Chosen file name and location is ignored when exporting clippings

### ISSUE THREAD

**Issue by UrsBeeli** [aecreations/clippings#184]
**Summary**
When trying to export my clippings, the file name and location of the file is not used. No matter what you choose, the export is always written to the default location and default file name.

**Steps to reproduce:**
1) open Clippings Manager
2) Tools -> Export...
3) "Clippings 6" & "Export"
--> a "save as" dialog pops up, initial location is "<UserName>\Downloads" and initial filename is "clippings.json"
4) change directory to some other place (i.e. F:\Clippings)
5) change file name to something else (i.e. "clipping-backup-2018-02-25.json")
6) "Save"

**Observed behaviour:**
No new file is written to "F:\Clippings".
Going to "<UserName>\Downloads" a new file "clippings.json" is created

**Expected behaviour:**
The export should be written to "F:\Clippings\clipping-backup-2018-02-25.json"
Nothing should be written to "<UserName>\Downloads". An existing "clippings.json" must not be overwritten.

**Comment by aecreations** [comment]
Does this issue happen when you try to save to a different folder on the local hard drive (C:)?  Is drive F: a network drive or an external storage device?

**Comment by UrsBeeli** [comment]
I picked "F:"  as an arbitrary driver other than the default one. In my case it was actually D:\.... which is my second local hard drive (C: is an SSD, D: is a large, classical hard drive).

I just re-ran a test saving to C:\temp\some-driectory\my-clippings.json.

While the confirmation message still claimed that it had succesfully saved "clippings.json", the file was actually written to the correct location with the correct name.

It seems that this only breaks when a change in volumen (drive letter) is involved. However, as that is exactly my use case (I keep all my data on D:, not on C:) I would still hope to see this fixed.

**Comment by aecreations** [comment]
Indeed, the confirmation message is misleading, as it shows the default export filename, rather than what was specified in the Save As dialog. This is being dealt with in a separate issue, #186.

As for the breakage when attempting to save to a file location with a different drive letter, is this issue also occurring when trying to save a download (e.g. an email attachment from Gmail, Outlook.com, etc.)?  The reason I ask is because Clippings is leveraging Firefox's download manager when saving the export file.

**Comment by UrsBeeli** [comment]
No, this issue does not occur when saving other downloads from within firefox. I have succesfully been saving downloads to my D: drive using firefox both pre and post Quantum.
